packages/i18n/README.md

```diff
@@ -0,0 +1,26 @@
+# JSONForms - More Forms. Less Code
+### Complex Forms in the blink of an eye
+
+JSONForms uses HTML custom elements and eliminates the need to write HTML templates in order to create forms by leveraging the capabilities of JSON and JSON schema.
+
+# I18n Package
+This is the JSONForms i18n package. This package provides reducers and actions to support i18n.
```

Comment:
Off-topic, but I'd propose to make this package private for the 2.0.0 release as it hasn't been tested thoroughly and it might be the case that we need to update the API.

Reply:
@eneufeld Do you agree?

Reply:
agreed

Reply:
Ok, I'll submit a PR later today

Reply:
+1